iothub/service/src/DigitalTwin/DigitalTwinClient.cs

```diff
@@ -0,0 +1,134 @@
+
+        /// <summary>
+        /// Updates a digital twin asynchronously.
+        /// <para>For further information on how to create the json-patch, see <see href="https://docs.microsoft.com/en-us/azure/iot-pnp/howto-manage-digital-twin."/></para>
+        /// </summary>
+        /// <param name="digitalTwinId">The Id of the digital twin.</param>
+        /// <param name="digitalTwinUpdateOperations">The application/json-patch+json operations to be performed on the specified digital twin.</param>
+        /// <param name="requestOptions">The optional settings for this request.</param>
+        /// <param name="cancellationToken">The cancellationToken.</param>
+        /// <returns>The http response.</returns>
+        public Task<HttpOperationHeaderResponse<DigitalTwinUpdateHeaders>> UpdateAsync(string digitalTwinId, string digitalTwinUpdateOperations, DigitalTwinUpdateRequestOptions requestOptions = default, CancellationToken cancellationToken = default)
+        {
+            return _protocolLayer.UpdateDigitalTwinWithHttpMessagesAsync(digitalTwinId, digitalTwinUpdateOperations, requestOptions?.IfMatch, null, cancellationToken);
+        }
+
+        /// <summary>
+        /// Invoke a command on a digital twin asynchronously.
+        /// </summary>
+        /// <param name="digitalTwinId">The Id of the digital twin.</param>
+        /// <param name="commandName">The command to be invoked.</param>
+        /// <param name="payload">The command payload.</param>
+        /// <param name="requestOptions">The optional settings for this request.</param>
+        /// <param name="cancellationToken">The cancellationToken.</param>
+        /// <returns>The application/json command invocation response and the http response. </returns>
+        public async Task<HttpOperationResponse<DigitalTwinCommandResponse, DigitalTwinInvokeCommandHeaders>> InvokeCommandAsync(string digitalTwinId, string commandName, string payload, DigitalTwinInvokeCommandRequestOptions requestOptions = default, CancellationToken cancellationToken = default)
+        {
+            using HttpOperationResponse<string, DigitalTwinInvokeRootLevelCommandHeaders> response = await _protocolLayer.InvokeRootLevelCommandWithHttpMessagesAsync(digitalTwinId, commandName, payload, requestOptions?.ConnectTimeoutInSeconds, requestOptions?.ResponseTimeoutInSeconds, null, cancellationToken)
+                .ConfigureAwait(false);
+            return new HttpOperationResponse<DigitalTwinCommandResponse, DigitalTwinInvokeCommandHeaders>
+            {
+                Body = new DigitalTwinCommandResponse { Status = response.Headers.XMsCommandStatuscode, Payload = response.Body },
+                Headers = new DigitalTwinInvokeCommandHeaders { RequestId = response.Headers.XMsRequestId },
+                Request = response.Request,
+                Response = response.Response,
+            };
+        }
+
+        /// <summary>
+        /// Invoke a command on a digital twin asynchronously.
```

Comment:
probably reword to 'Invoke a command on a digital twin component asynchronously.'

Reply:
will do!